src/seahawk/seahawk_deck/set_remote_params.py

```diff
@@ -0,0 +1,85 @@
+"""
+set_remote_params.py
+
+Abstracts requesting to set another node's parameters via service to class
+'SetRemoteParams'. To import this class to a program on the deck use 
+'from .set_remote_params import SetRemoteParams'
+
+Copyright (C) 2022-2023 Cabrillo Robotics Club
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+Cabrillo Robotics Club
+6500 Soquel Drive Aptos, CA 95003
+cabrillorobotics@gmail.com
+"""
+# ROS client library
+from rclpy.node import Node 
+
+# Type hints
+from typing import Any
+
+# Parameters
+from rcl_interfaces.srv import SetParameters
+from rclpy.parameter import Parameter
+
+class RemoteParamsClient():
+    """
+    Abstracts requesting to set another node's parameters via service to a class. 
+    """
+    def __init__(self, this_node: Node, other_node_name: str):
+        """
+        Set up request to set another node's parameters via service named '/other_node_name/set_parameters'
+    
+        Args:
+            this_node: Node which makes the request to set another node's parameters. Client node
+            other_node_name: Name of the node whose parameters should be set. Service node
+        """
+        # Create service name. Services to set parameters must be named as '/other_node_name/set_parameters'
+        srv_name = "/" + other_node_name + "/set_parameters"
```

Comment:
I think it would be somewhat "more correct" to not have the prefixing `/` as there is a sort of namespace hierarchy in ROS.

Reply:
Services are always in the form `/node_name/srv_name`
![services](https://github.com/CabrilloRoboticsClub/cabrillo_rov_2023/assets/96219204/8597de08-f854-4cb4-b864-b98d0c10bcf3)

Reply:
Yes, but think of it like absolute paths versus relative paths. ROS will prefix it with whatever it needs.
At least, that's my interpretation of how it works.